.travis.yml

```diff
@@ -21,7 +20,7 @@ env:
   matrix:
     - JHIPSTER=app-from-scratch JHIPSTER_NODE_CACHE=0 RUN_APP=0
     - JHIPSTER=app-gradle RUN_APP=0
-    - JHIPSTER=app-gulp GRUNT=0
+    - JHIPSTER=app-gulp
```

Comment:
@pascalgrimaud this is now same as app-default, so either we can remove this or add something useful instead